content/guides/kubernetes/ytt-gs.md

```diff
@@ -0,0 +1,269 @@
+---
+date: '2021-08-16'
+lastmod: '2021-08-16'
+parent: Carvel
+patterns:
+- Deployment
+tags:
+- ytt
+- Carvel
+team:
+- Leigh Capili
+- Tiffany Jernigan
+title: Getting Started with ytt
+topics:
+- Kubernetes
+oldPath: "/content/guides/kubernetes/ytt-gs.md"
+aliases:
+- "/guides/kubernetes/ytt-gs"
+level1: Deploying Modern Applications
+level2: CI/CD, Release Pipelines
+---
+
+[ytt](https://carvel.dev/ytt/) (part of the open source [Carvel](https://carvel.dev) suite) is a tool that can templatize, combine, and patch any YAML content.
+ytt understands the structure of YAML. 
+This eliminates the need to count spaces or escape quotes in traditional text-based templating.
+ytt uses a sandboxed, Pythonic language, embedded within YAML comments, to provide a natural, programming extension for YAML.
+
+ytt can simplify and variate any YAML content, including Kubernetes configuration, Concourse pipelines, Docker Compose files, and more.
+
+This guide will cover the basics of using ytt to configure Kubernetes apps.
+## Prerequisites
+Before you get started you will need to do the following:
+* Create a Kubernetes cluster
+* Install `kubectl` locally
+* Install the `ytt` CLI via one of these options:
+* [Homebrew](https://github.com/vmware-tanzu/homebrew-carvel): 
+    ```
+    brew tap vmware-tanzu/carvel
+    brew install ytt
+    ```
+	* [GitHub releases](https://github.com/vmware-tanzu/carvel-ytt/releases/tag/v0.36.0): move it to `/usr/local/bin` or add it to your `$PATH` and run `chmod +x` to make it executable
```

Comment:
```suggestion
   * [GitHub releases](https://github.com/vmware-tanzu/carvel-ytt/releases/tag/v0.36.0): move it to `/usr/local/bin` or add it to your `$PATH` and run `chmod +x` to make it executable
```

Reply:
From the VMware style guide, lists formatted like this need to be separated by a "--" as opposed to a ":"

Reply:
That is extremely specific...wow.